core/servers/chat/mrc_multiplexer.js

```diff
@@ -245,8 +245,7 @@ exports.getModule = class MrcModule extends ServerModule {
         connectedSockets.forEach(client => {
             if (
                 message.to_user == '' ||
-                // Fix PrivMSG delivery on case mismatch
-                message.to_user.toUpperCase() == client.username.toUpperCase() ||
+                message.to_user == client.username.toUpperCase() ||
```

Comment:
These would technically be better using [localeCompare](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare) with sensitivity 'accent'.